PERMISSIONS.md

```diff
@@ -0,0 +1,53 @@
+| **[nodejs/nodejs.org-archive][]**        | Read     | Write       | -                      | Maintain          | Admin              | This repository is _archived_. |
+| **[nodejs/release-cloudflare-worker][]** | Read     | -           | -                      | Maintain          | Admin              |                                |
+| **[nodejs/web-team][]**                  | Read     | Triage      | Maintain               | -                 | Admin              |                                |
+| **[nodejs/website-cloudflare-worker][]** | -        | -           | -                      | Maintain          | Admin              |                                |
+
+## External Services
+
+| Service                   | Everyone | @nodejs/web | @nodejs/nodejs-website | @nodejs/web-infra | @nodejs/web-admins | Notes                                                                                                                      |
+| ------------------------- | -------- | ----------- | ---------------------- | ----------------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------- |
+| **[1Password][]**         | -        | -           | -                      | Admin             | Admin              |                                                                                                                            |
+| **[Chromatic][]**         | Read     | -           | Write                  | Admin             | Admin              | Access to this service is granted via GitHub authentication.                                                               |
+| **[Cloudflare][]**        | -        | -           | -                      | Read              | Admin              | Access to this service is controlled by @nodejs/build. Additional access may be granted on a case-by-case basis.           |
+| **[Codecov][]**           | Read     | -           | Admin                  | Admin             | Admin              | Access to this service is granted via GitHub. authentication Access to this service only covers repositories listed above. |
+| **[Figma Design File][]** | Read     | -           | Write                  | -                 | -                  | Access to this service is controlled by the OpenJS Foundation and @avivkeller.                                             |
+| **[Sentry][]**            | -        | -           | -                      | Admin             | Admin              |                                                                                                                            |
+| **[Vercel][]**            | -        | -           | -                      | Admin             | Admin              |                                                                                                                            |
```

Comment:
Access to this service is also granted by a shared GitHub account

Reply:
Besides of individual access

Reply:
No, there's a shared dedicated login in 1Password that we can use for elevated access